CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v1.0.0-beta2 (unreleased)
+
+* [Breaking] Rename `modules/rpc` to `modules/yarpc`
```

Comment:
The key should be changed to yarpc. @sectioneight you need to just remove the default naming in the `ThriftModule` function (where it sets the name to "rpc"), see my comment on yarpc.go